utilities/private/isplottingfunction.m

```diff
@@ -0,0 +1,25 @@
+function [status] = isplottingfunction(function_name)
+
+% ISPLOTTINGFUNCTION is a helper function for reproducescript, and
+% is used for the cfg.reproducescript functionality. It compares the input
+% function name with the list of known FieldTrip plotting functions and
+% returns 1 if it is a plotting function, and 0 otherwise.
+
+% determine FieldTrip path
+ftpath = fileparts(which('ft_defaults'));
+
+functionlist = [];
+% browse the main FieldTrip directory
+functionlist = [functionlist; dir([ftpath,'/', '*plot*.m'])];
```

Comment:
these types of commands are not guaranteed to be robust for all operating systems. Windows has a '\' rather than a '/'. It could be that more recent versions of matlab know how to deal with this, but more robust would be to use fullfile, rather than the [somestring, '/', someotherstring] syntax

Reply:
Note that I took the liberty of making this change directly into this PR

Reply:
there is also the MATLAB function `filesep`

Reply:
I know, but that leads to ugly code

Reply:
don't forget strcat